grpc/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Can you please rename the script to "just" `run`? This way it will nicely co-exists next to the other existing run scripts. See: https://github.com/IBM/CodeEngine/blob/main/bin/run-all